mParticle-Example/MPKitResponsys.m

```diff
@@ -0,0 +1,347 @@
+//
```

Comment:
The intention of `Example` in mParticle-Example was to be replaced by the company name that's implementing the kit, rather than as an actual Example project. However, I can see how that would be confusing.

Also it's helpful to have the other CocoaPods project as an example, so thanks for adding that.

This Xcode project is actually used to provide Carthage support via `carthage update; carthage build --no-skip-current; carthage archive`, which results in a `.framework.zip` file that will be attached to GitHub releases.